app/code/Bulbulatory/Recomendations/Model/Recommendation.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace Bulbulatory\Recomendations\Model;
+
+class Recommendation extends \Magento\Framework\Model\AbstractModel implements \Magento\Framework\DataObject\IdentityInterface
+{
+    const CACHE_TAG = 'bulbulatory_recomendations_recommendation';
+    protected $_cacheTag = 'bulbulatory_recomendations_recommendation';
+    protected $_eventPrefix = 'bulbulatory_recomendations_recommendation';
+
+    protected function _construct()
+    {
+        $this->_init('Bulbulatory\Recomendations\Model\ResourceModel\Recommendation');
```

Comment:
Gdy potrzebna jest nazwa klasy, lepiej użyć `::class` niż podać nazwę klasy jako string.
```suggestion
        $this->_init(Bulbulatory\Recomendations\Model\ResourceModel\Recommendation::class);
```
PHP sam sobie to zamieni na klasę jako string, ale sama praca jest wygodniejsza, ponieważ w związku z tym, że nie operujemy na stringach, wiele IDE będzie oferowało np. inne formatowanie takiego kawałka kodu albo w przypadku zmiany nazwy klasy automatycznie podmieni wszystkie jej wystąpienia. Jeśli będziemy chcieli sprawdzić, co to za klasa, to w wielu edytorach także będzie można bezpośrednio do niej przejść z tego miejsca, a nie ręcznie wyszukiwać.